docs/topics/get-logs.md

```diff
@@ -0,0 +1,47 @@
+# Retrieving node and cluster logs
+
+## Prerequisites
+
+All the commands in this guide require both the Azure CLI and `aks-engine`. Follow the [quickstart guide](../tutorials/quickstart.md) before continuing.
+
+This guide assumes you already have deployed a cluster using `aks-engine`. For more details on how to do that see [deploy](../tutorials/deploy.md).
+
+## Retrieving Logs
+
+The `aks-engine get-logs` command can be useful to troubleshoot issues with your cluster. It will produce, collect and download to your workstation a set of files that include node configuration, cluster state and configuration, and provision log files.
+
+At a high level, it works by stablishing a SSH session into each node, executing a `log collection` script that collects and zips relevant files, and downloading the zip file to your local computer.
+
+### SSH Sessions
+
+A valid SSH private key is always required to stablish a SSH session to the cluster Linux nodes. Windows credentials are stored in the apimodel and will be loaded from there.
```

Comment:
Do you want to add a comment that says SSH needs to be enabled on Windows nodes by setting windowsProfile.sshEnabled = true?

(I opened a PR to default ssh to be enabled for Windows but it went stale after nobody reviewed it  :()

Reply:
https://github.com/Azure/aks-engine/blob/master/docs/topics/clusterdefinitions.md#windowsprofile